packages/date-picker/src/FilterDateRangePicker/components/DateRangeInputField/DateRangeInputField.tsx

```diff
@@ -28,11 +28,12 @@ export interface DateRangeInputFieldProps
   /**
    * Updates the styling of the validation FieldMessage
    */
-  status?: FieldMessageStatus
+  status?: { dateStart?: FieldMessageStatus; dateEnd?: FieldMessageStatus }
```

Comment:
Separate the `dateStart` and the `dateEnd` status due to ability to have separate validation

Reply:
Nice, but I do wonder what we expect the consumer to do when:
- There is an error for the field as a whole (you'd want both inputs highlighted)
- Both inputs have different errors and require different validation messages

Thoughts?

Reply:
> Both inputs have different errors and require different validation messages

I'd argue that this would be an edge case since if both start/end are invalid, what are the chances that they'd need separate messages if the collective data is invalid they'd use a collective message?

An example scenario could be that the start/end dates are too far apart, and the message would be "Please enter dates which are closer together".

Reply:
I will be raising this with @jmitsios tomorrow for a designers perspective

Reply:
I can see how we could use this api as it is to solve it (get the consumer to figure it out), but it's worth thinking about anyway 😉 

However, when we put in the inbuilt validation, we might have someone type gibberish in one input, then type a disabled date with the other input, so we'll end up with different errors for each input. How should we go about piecing them together? Do we only show one message at a time?